app/src/main/java/com/example/lister/ListDatabaseHelper.java

```diff
@@ -219,6 +219,25 @@ public void updateItemPrice(double itemPrice, int id){
                 ListerDatabase.Item._ID + "= " + id, null);
     }
 
+    public String getItemPrice(int id){
+        SQLiteDatabase db = getReadableDatabase();
+        String LIST_QUERY = String.format("SELECT itemPRICE FROM %s WHERE %s=%s", ListerDatabase.Item.TABLE_NAME, ListerDatabase.Item._ID,id);
```

Comment:
I think we can call the ListerDatabase.Item.price or whatever it is instead of itemPRICE here?